.github/bot-files/labels.yml

```diff
@@ -0,0 +1,24 @@
+## Defines the list of labels to apply to a PR when there is a change in a specified path
+# the default label is applied to all the PRs, other labels will filter based on the files changed on the PR.
+# follows the same pattern as reviewers.yml.
+# please see the example for the file below
+
+labels:
```

Comment:
Don't forget to add the emoticons to the labels, otherwise won't work.

Reply:
Those things are needed to change in the bot code base, will add these features later.

Reply:
>  After all the CI is passed and all the reviewers have approved the PR?

This ^

Reply:
Are these two features of priority, or can be added later? (I mean after having this much so we can know what other things are also needed or something that needs to be changed.)

Reply:
Lets merge and work with what we got, open JIRAs to track this new features and work on them thorough the sprint. Don't push yourself :)